Sources/TestSupport/DependencyResolverHelper.swift

```diff
@@ -0,0 +1,115 @@
+/*
```

Comment:
I'd probably just call this `MockDependencyResolver` and put that here, to start (and the XCTAssert)